### PR TITLE
support enable/disable vacuum

### PR DESCRIPTION
**Pull request by guol-fnst**
# What problem are we solving?
Make vacuuming more flexible.
If the cluster is in heavy load or it's not the proper time to peform vacuuming.
We can used volume.suspendvacuum to suspend vacuuming and use volume.resumevacuum to continue vacuuming

# How are we solving the problem?

# How is the PR tested?

# Checks
- [ ] I have added unit tests if possible.
- [ ] I will add related wiki document changes and link to this PR after merging.

### CODE REVIEW

File: weed/shell/command_volume_resume_vacuum.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+func init() {
+	Commands = append(Commands, &commandResumeVacuum{})
+}
+
+type commandResumeVacuum struct {
+}
+
+func (c *commandResumeVacuum) Name() string {
+	return "volume.resumevacuum"
```
Comment (chrislusf) [inline weed/shell/command_volume_resume_vacuum.go]: This "resume" does not start from where the vacuum is paused. It should not be called "resume".

how about using `volume.vaccum.enable` and `volume.vaccum.disable`? 

And also change the file names and variable names accordingly.

Comment (guol-fnst) [reply weed/shell/command_volume_resume_vacuum.go]: Done!

File: weed/topology/topology.go [hunk 2]
```diff
@@ -35,6 +35,7 @@ type Topology struct {
 
 	volumeSizeLimit  uint64
 	replicationAsMin bool
+	VacuumStatus     bool
```
Comment (chrislusf) [inline weed/topology/topology.go]: rename this

Comment (guol-fnst) [reply weed/topology/topology.go]: How about renaming it  to "isDisableVacuum"

Comment (guol-fnst) [reply weed/topology/topology.go]: Done

**Comment by guol-fnst** [comment]
@chrislusf  Does this PR need more work?

**Pull request merged**